Clean up documentation (#1506)
jmorganca/ollama: docs/faq.md

# FAQ

> ## How can I upgrade Ollama

Reviewer: ```suggestion
## How can I upgrade Ollama?
```

> To upgrade Ollama, run the installation process again. On the Mac, click the Ollama icon in the menubar and choose the restart option if an update is available.

Reviewer: ```suggestion
To upgrade Ollama, run the installation process again. On the macOS, click the Ollama icon in the menubar and choose the restart option if an update is available.
```

Author: on macOS, not the macOS

## How can I view the logs?

Review the [Troubleshooting](./troubleshooting.md) docs for more about using logs.

## How do I use Ollama server environment variables on Mac

On macOS, Ollama runs in the background and is managed by the menubar app. If adding environment variables, Ollama will need to be run manually.

1. Click the menubar icon for Ollama and choose **Quit Ollama**.
2. Open a new terminal window and run the following command (this example uses `OLLAMA_HOST` with an IP address of `123.1.1.1`):

   ```bash
   OLLAMA_HOST=123.1.1.1 ollama serve
   ```

> ## How do I use Ollama server environment variables on Linux

Reviewer: ```suggestion
## How do I use Ollama server environment variables on Linux?
```

# FAQ

## How can I upgrade Ollama

To upgrade Ollama, just run the installation process again. On the Mac, you can click the Ollama icon in the menubar and choose the restart option if an update is available.

## How can I view the logs?

Review the [Troubleshooting](./troubleshooting.md) docs for more about using logs.

## How can I expose Ollama on my network?

To expose Ollama to another host, you need to use the `OLLAMA_HOST` environment variable. See [Serving Ollama in the CLI Documentation](./cli.md#serving-ollama) for more on using this.

## How can I allow additional web origins to access Ollama?

You will need to use `OLLAMA_ORIGINS` to allow additional web origins. See [Serving Ollama in the CLI Documentation](./cli.md#serving-ollama) for more on using this.

## Where are models stored?

> - macOS: All model data is stored under `~/.ollama/models`.
> - Linux: All model data is stored under `/usr/share/ollama/.ollama/models`

Reviewer: ```suggestion
- macOS: `~/.ollama/models`.
- Linux: `/usr/share/ollama/.ollama/models`
```

See [the CLI Documentation](./cli.md) for more on this.

## Does Ollama send my prompts and answers back to Ollama.ai to use in any way?

> No, Ollama runs 100% locally wherever you have installed it. Other than downloading the models, we never reach out anywhere, and models are never updated using your questions and answers.

Reviewer: ```suggestion
No, Ollama runs entirely locally, and conversation data will never leave your machine.
```

# FAQ

> 

Reviewer: The FAQ is still the right spot for common questions.

Author: We had discussed in standup removing most questions from faq and moving them to more appropriate places in docs
